app/api.py

```diff
@@ -799,16 +799,26 @@ def get_element_text(self, doc, tag_name):
         else:
             return None
 
-    # ga_send_hit sends hit to Google Analytics via Measurment Protocol.
-    # With Measurement Protocol, you can send data by making HTTP requests.
-    # You can find more on developer guide.
-    # https://developers.google.com/analytics/devguides/collection/protocol/v1/devguide
-    def ga_send_hit(self, event_category):
-        ga_cid = uuid.uuid4()
+    def send_hit_to_google_analytics(self, event_category):
+        """Sends hit to Google Analytics via Measurment Protocol.
+        With Measurement Protocol, you can send data by making HTTP requests.
+        You can find more on developer guide.
+        https://developers.google.com/analytics/devguides/collection/protocol/v1/devguide
+        Args:
+            event_category: Google Analytics Event Category.
+        """
+
+        # Check Google Analytics property setting
```

Comment:
This comment is too obvious, so you won't need it.

In general, it's recommended to add a document for each method/class, but otherwise you should write comment only if it's not obvious what the code does or why such code is there.